### ISSUE TITLE
insufficient contrasts on pagination focus

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#397]
Standard pagination items, when focused, are not contrasted enough: their `border` is the same as the `:hover` one, which is only `#cccccc` on `#ffffff`.

I recommend adding a `#000000` solid `outline` alongside `border`
